experimenter/experimenter/settings.py

```diff
@@ -344,6 +344,17 @@
 REDIS_PORT = config("REDIS_PORT")
 REDIS_DB = config("REDIS_DB")
 
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+    },
```

Comment:
We can probably leave 'default' out here altogether since we're not using caching anywhere and declaring it here could indicate that we expect it to be available when we don't.

Reply:
So `default` is required, which means we can either use the Redis cache as the default, or do something like what I have here. My original thought was to leave the default as-is, but if we are intentionally not using it then I think it's fine to swap it for the Redis one (and remove the `sizing` cache).